packages/@atjson/source-ckeditor/test/source-classic-build-test.ts

```diff
@@ -1,19 +1,7 @@
-import { Annotation } from "@atjson/document";
+import { compareAnnotations } from "@atjson/document";
```

Comment:
Instead of using `compareAnnotations` here, could we change the call sites from:

```ts
doc.annotations.sort(compareAnnotations)
```

to:

```ts
doc.all().sort().annotations
```

Reply:
That's understandable!